src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -30,6 +30,7 @@ def __init__(
         super().__init__(sink_config.robusta_sink)
         self.token = sink_config.robusta_sink.token
         self.cluster_name = cluster_name
+        self.account_id = account_id
```

Comment:
I would use the account_id from the token.
It's safer to assume the token has the correct one.